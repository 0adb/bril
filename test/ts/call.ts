var x : number = 5;
var y : number = 2;
//var b = add2(x, y);
//console.log(y)

function add2(x : number, y : number) : number {
    return x + 1;
}