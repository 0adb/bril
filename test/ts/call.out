main {
  v0: int = const 5;
  x: int = id v0;
  v1: int = id x;
  call v1;
  v2: int = const 0;
}
callPrint {
  v0: int = id x;
  print v0;
  v1: int = const 0;
}
